endpoints/CreateRoomEndpoint.ts

```diff
@@ -74,24 +95,36 @@ export class PushEndpoint extends ApiEndpoint {
 
         visitor.visitor.department = departmentId;
 
-        const createdVisitor = RocketCaller.createVisitor(read, http, visitor);
-        if (!createdVisitor) {
-            const errorMessage = `Could not create visitor: ${visitor}`;
+        const createdVisitor = await rocketUtils.createVisitor(visitor);
+        if (!createdVisitor || createdVisitor.statusCode !== 200) {
+            const errorMessage = `Could not create visitor: ${visitor}\nResponse obtained: ${createdVisitor}`;
             this.app.getLogger().error(errorMessage);
             return httpErrorResponse(HttpStatusCode.BAD_REQUEST, errorMessage);
         }
 
-        const createdRoom = RocketCaller.createRoom(read, http, token, priority);
-        if (!createdRoom) {
-            const errorMessage = `Could not create room for visitor with token: ${token}`;
+        const createdRoom = await rocketUtils.createRoom(token, priority);
+        if (!createdRoom || createdRoom.statusCode !== 200) {
+            const errorMessage = `Could not create room for visitor with token: ${token}\nResponse obtained: ${createdRoom}`;
             this.app.getLogger().error(errorMessage);
             return httpErrorResponse(HttpStatusCode.BAD_REQUEST, errorMessage);
         }
 
         const after = msgsAfter ? msgsAfter : getNowDate();
```

Comment:
Add TODO to check if the `getNowDate` comes right in the production environment.